test/credentials_test.js

```diff
@@ -40,3 +42,28 @@ suite('pulseCredentials', function() {
       'amqps://ali-escaper:/@%5C%7C()%3C%3E&:bobby-tables:/@%5C%7C()%3C%3E&@pulse.abc.com:5671/%2F');
   });
 });
+
+suite('claimedCredentials', function() {
+  test('missing arguments are an error', async function() {
+    assume(() => claimedCredentials({credentials: {}, namespace: 'ns'}))
+      .throws(/rootUrl/);
+    assume(() => claimedCredentials({rootUrl: 'rU', namespace: 'ns'}))
+      .throws(/credentials/);
+    assume(() => claimedCredentials({rootUrl: 'rU', credentials: {}}))
+      .throws(/namespace/);
+  });
+  
+  test('calls Pulse.claimNamespace to claim a namespace', async function() {
+    nock('http://fake')
+      .post('/api/pulse/v1/namespace/ns')
+      .reply(200, {'namespace': 'ns', 'connectionString': 'cs'});
```

Comment:
there won't be quotes in namespace and connectionString

Reply:
That will be encoded as JSON (in this case by `nock`, and in the real case by `express`), and then decoded by `taskcluster-client`.  So the input here is just a JS data structure, and the output will be a matching data structure.

Reply:
Oh, I see what you mean -- there *shouldn't* be quotes.  Yes, you're absolutely right :)